scss/utilities/mixins/molecules/index.scss

```diff
@@ -0,0 +1,13 @@
+@charset "UTF-8";
```

Comment:
Why the start of a new convention here? Why nest these elements in a molecules directory?